README.md

```diff
@@ -220,12 +216,17 @@ To understand what we're trying to become, it may be useful to understand what
 we're trying to avoid. See effects of [contempt](http://blog.aurynn.com/86/contempt-culture)
 and [competition](https://medium.com/tech-diversity-files/if-you-think-women-in-tech-is-just-a-pipeline-problem-you-haven-t-been-paying-attention-cb7a2073b996#.yjrp4mqkb) on environments.
 
-#### Favour experimentation over argument
+#### Favour experimentation over opinion
 
 #### Be excellent to each other!
 
 ### Be Entrepreneurial
 
+#### Build Minimum Viable Products (MVPs)
+- When beginning a new feature with a high amount of ambiguity or uncertainty about whether it will really be a hit, think "What is the MVP for this" that is, what is the minimum amount of work we can do to deliver the value of this feature to the user.
+- In other words: Don't gold plate before validating your idea.
+- Collect some meaningful metric(s) to measure it's effectiveness. Hint: product managers are excited to talk about KPIs!
```

Comment:
So, this *could* belong here, but I like this comment better under Customers First because it puts a greater emphasis on ensuring we're focusing on customers when building MVPs. Open to counterarguments though :)